src/helpers.js

```diff
@@ -169,28 +169,6 @@ const helpers = {
     return version.replace(/[^\d.]/g, '').split('.');
   },
 
-  /**
-   * Get the server URL that the wallet is connected
-   *
-   * If a server was not selected, returns the default one
-   *
-   * @return {string} Server URL
-   *
-   * @memberof Helpers
-   * @inner
-   */
-  getServerURL() {
```

Comment:
This method is used by the wallets. I think every method and constant we are removing should be checked in all the wallets

Reply:
I'll make it a proxy for the new method